views/partials/header.njk

```diff
@@ -1,53 +1,45 @@
 
-        {% if user %}
-            <div class="right-content">
-                <div class="right-menu no-select">
-                    <label class="right-menu-content clickable" for="menuToggle">
-                        <input id="menuToggle" type=checkbox>
-                        <div id="info-wrapper" class="info-wrapper right-menu-info-wrap" role="menu" tabindex="0">
-                            <strong class="user-name right-menu-content-primary">{{ user.displayName }}</strong>
-                            <span class="case-load right-menu-content-secondary">{{ user.activeCaseLoad.description }}</span>
-                        </div>
-                        <div class="right-menu-dropdown-menu" id="menuItems">
-                            <div>
-                            {% if user.allCaseloads.length > 1 %}
-                                <a class="right-menu-dropdown-menu-option" href="/change-caseload">Change caseload</a>
-                            {% endif %}
-                            <a class="right-menu-dropdown-menu-option" href="/auth/logout">Sign out</a>
-                            </div>
-                        </div>
-                    </label>
-                </div>
-            </div>
-        {% endif %}
+        <image src="/assets/images/moj-logotype-crest.png" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
+      </svg>
+
+      <a class="dps-header__link dps-header__title__organisation-name" href="/">HMPPS</a>
+
+      <a class="dps-header__link dps-header__title__service-name" href="/">Digital prison services</a>
     </div>
+
+    <nav aria-label="Account navigation">
+      <ul class="dps-header__navigation">
+        <li class="dps-header__navigation__item">
+          <a class="dps-header__link" href="{{ authUrl + 'account-details' }}">
+            {{ user.displayName }}
+            <span class="dps-header__link__sub-text">Manage your details</span>
```

Comment:
Why is this _Manage your details_ instead of _Manage account details_ in auth?  Should we change the auth one to match?

Reply:
Just double checked with Eddie, will change to _Manage account details_